lib/services/lookup.js

```diff
@@ -198,6 +198,10 @@ function lookupServiceFactory(
         };
     };
 
+    LookupService.prototype.reqIpAddressToMacAddress = function (req) {
```

Comment:
that's needed for the 2.0 templates route, I figured I'd get the dependency satisfied now, so that I can just submit a single PR for templates.